### DISCUSSION TITLE
Why it always cache miss?

### DISCUSSION THREAD

**Discussion by kiic-leung** [verdaccio/verdaccio#3432] [Q&A] [answered]
I use verdaccio as a proxy cache  for the public npm repo.

And run npm install command several times, but it always show "cache miss" but not "cache hit".

_npm http fetch GET 200 http://localhost:4873/react-syntax-highlighter 50ms (cache miss)_

**Comment by juanpicado** [comment, marked as answer]
That's an npm client topic no verdaccio, my friend, you should ask them directly https://github.com/npm/cli#links--resources

If I knew I'd write an answer but I don't sorry.
